pgsql-flexible-server.tf

```diff
@@ -72,3 +84,14 @@ resource "azurerm_postgresql_flexible_server_configuration" "pgsql_server_config
   server_id = azurerm_postgresql_flexible_server.pgsql_server.id
   value     = each.value.value
 }
+
+
+resource "azurerm_postgresql_flexible_server_active_directory_administrator" "pgsql_adadmin" {
+  count               = var.set_ad_admin != null ? 1 : 0
+  server_name         = azurerm_postgresql_flexible_server.pgsql_server.name
+  resource_group_name = azurerm_postgresql_flexible_server.pgsql_server.resource_group_name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  object_id           = data.azurerm_client_config.current.object_id
+  principal_name      = var.ad_pricipal_name
+  principal_type      = var.ad_principal_type
+}
```

Comment:
Can we do this more like cnp-module-postgres please?

https://github.com/hmcts/cnp-module-postgres/blob/postgresql_tf/main.tf#L71-L91

if you need to add a method for overriding but it shouldn't be the default

Reply:
updated it now, please check.